### PR TITLE
feat(dropdown): HTML & CSS structure of new nested dropdown style

### PR DESCRIPTION
**Pull request by pavlovicnemanja92**
This is relative to SD-2101 task in JIRA. This is complete HTML and CSS structure, and I provide a little  jQuery code that will need to be implement into angular js file relative to dropdown menus.

Also, I create this with dummy data, and as we speak on today's scrum, someone need to create dynamic structure for this.

Currently, I put all the HTML and jQuery code in metadata-terms.html, and it shows on every sdTypeaHead component in right side bar. We will need this only on subject field.

### CODE REVIEW

**Comment by amagdas** [comment]
:+1:

File: client/app/scripts/superdesk-authoring/metadata/views/metadata-terms.html [hunk 1]
```diff
@@ -1,12 +1,32 @@
 <div class="term-editor data visible">
-	<div sd-typeahead items="terms" term="selectedTerm" search="searchTerms(term)" select="selectTerm(item)" data-disabled="disabled">
-		<ul>
-			<li typeahead-item="t" ng-repeat="t in terms">{{ t.name }}</li>
-		</ul>
-	</div>
+    <div sd-typeahead items="terms" term="selectedTerm" search="searchTerms(term)" select="selectTerm(item)" data-disabled="disabled">
+        <ul>
+            <li typeahead-item="t" ng-repeat="t in terms ">{{ t.name }}</li>
```
Comment (plamut) [inline client/app/scripts/superdesk-authoring/metadata/views/metadata-terms.html]: Can we use one-time binding here? The name of each term does not change?

File: client/app/scripts/superdesk-authoring/metadata/views/metadata-terms.html [hunk 2]
```diff
@@ -1,12 +1,32 @@
...
+            <button ng-if="!tree[term.qcode]" ng-click="selectTerm(term)">{{ :: term.name }}</button>
+        </li>
+    </ul>
 </div>
 <div class="terms" ng-if="item[field].length">
-	<ul>
-		<li ng-repeat="t in item[field]" ng-click="removeTerm(t)">{{ t.name }}</li>
-	</ul>
+    <ul>
+        <li ng-repeat="t in item[field]" ng-click="removeTerm(t)">{{ t.name}}</li>
```
Comment (plamut) [inline client/app/scripts/superdesk-authoring/metadata/views/metadata-terms.html]: Similar question to above (`track by` and `::`)

**Comment by ioanpocol** [comment]
:+1:

**Comment by pavlovicnemanja92** [comment]
:+1:

**Pull request merged**